components/molecule/imageEditor/src/index.js

```diff
@@ -100,6 +104,7 @@ MoleculeImageEditor.propTypes = {
   cropLabelText: PropTypes.string,
   image: PropTypes.string.isRequired,
   onChange: PropTypes.func.isRequired,
+  onCropping: PropTypes.func,
```

Comment:
I would rename it to "onCrop" (since it is fired when crop is complete and not while cropping is being made)